bots/jabber_mirror_backend.py

```diff
@@ -195,13 +210,16 @@ def nickname_to_jid(self, room, nick):
 
 class ZulipToJabberBot(object):
     def __init__(self, zulip_client):
+        # type: (zulip.Client) -> None
         self.client = zulip_client
         self.jabber = None # type: JabberToZulipBot
 
     def set_jabber_client(self, client):
+        # type: (JabberToZulipBot) -> None
         self.jabber = client
 
     def process_event(self, event):
+        # type: (Dict[str, Any]) -> None
```

Comment:
I'm guessing that we can call `event` at least something as specific as `Dict[str, Any]`, correct?  Can you dig a bit further on what typical events will look like?